README.md

```diff
@@ -93,34 +93,18 @@ each file or in a readme file of the folder. The main other contributors are:
 
 # <a name="developers"></a>Developers Section
 
-## <a name="install-debian"></a>Installation on Debian/Ubuntu [OLD]
+## <a name="install-ubuntu"></a>Installation on Ubuntu 18.04 and 20.04 with a signed certificate using mkcert
 
 ```
-sudo add-apt-repository ppa:akretion-team/pywebdriver
-sudo apt-get update
-sudo apt-get install pywebdriver
+wget https://raw.githubusercontent.com/akretion/pywebdriver/master/debian/install.sh
+sudo chmod +x install.sh
+sudo ./install.sh
 ```
 
-## <a name="install-bionic"></a>Installation on Ubuntu 18.04 [NEW]
-
-```
-sudo apt-key adv --recv-keys --keyserver keyserver.ubuntu.com 305CB544DDFC7A97
-sudo apt-add-repository 'deb [arch=amd64] http://apt.akretion.com/ pywebdriver-nightly main'
-sudo apt update
-sudo apt install pywebdriver
-```
-
-If you update from the OLD version, you will need to uninstall nginx after upgrading or
+If you update from an OLD version, you will need to uninstall nginx after upgrading or
 you could face port reservation problems. Also it is not required anymore so there is no
 reason to have it.
 
-## <a name="install-script"></a>Installation on Ubuntu 18.04 with a signed certificate using mkcert
-
-Alternatively you can use the script
-[debian/install.sh](https://github.com/akretion/pywebdriver/tree/master/debian/install.sh).
```

Comment:
The script is still relevant, maybe the text should be changed but we should not just remove it

Reply:
Yes i keep it actually, I put together the two parts of the readme (for ubuntu) into one part, so there is only one way to install pywebdriver, with this script :)